sigmastate/src/test/scala/sigmastate/SoftForkabilitySpecification.scala

```diff
@@ -371,4 +378,24 @@ class SoftForkabilitySpecification extends SigmaTestingData {
       CheckCostFunc(tIR)(asRep[Any => Int](costF))
     })
   }
+
+  property("PrecompiledScriptProcessor is soft-forkable") {
+    val p = ErgoLikeTestInterpreter.DefaultProcessorInTests
+    val v1key = CacheKey(propV2treeBytes, vs)
+    checkRule(CheckValidOpCode, v2vs, {
+      p.getReducer(v1key)
+    })
+
+    val v2key = CacheKey(propV2treeBytes, v2vs)
+    val r = p.getReducer(v2key)
+    r shouldBe WhenSoftForkReducer
+
+    val ctx = createContext(blockHeight, txV2, v2vs)
+    r.reduce(ctx) shouldBe WhenSoftForkReductionResult
+  }
+
+  override protected def afterAll(): Unit = {
+    println(ErgoLikeTestInterpreter.DefaultProcessorInTests.cache.stats())
```

Comment:
Use logger instead of println

Reply:
@kushti There is no infrastructure for logging in sigma, and we never used logging in tests where as println is used everywhere from day one. I'm not sure we need it at all, println seems ok for tests.